### PR TITLE
Add testgrid page for pr-cloud-provider-e2e

### PR DESCRIPTION
**Pull request by feiskyer**
/assign @ritazh

### CODE REVIEW

File: testgrid/config.yaml [hunk 1]
```diff
@@ -7804,6 +7813,15 @@ dashboards:
   - name: pr-k8s-e2e
     description: Run e2e tests
     test_group_name: pull-kubernetes-e2e-aks-engine-azure
+  - name: pr-cloud-provider-e2e
+    description: Run Kubernetes e2e tests for cloud-provider-azure
+    test_group_name: pull-cloud-provider-azure-e2e
+  - name: pr-cloud-provider-unit
+    description: Run unit tests for cloud-provider-azure
+    test_group_name: pull-cloud-provider-azure-unit
+  - name: pull-cloud-provider-check
```
Comment (spiffxp) [inline testgrid/config.yaml]: Nit: seems like you name everything else in this dash with `pr-`
```suggestion
  - name: pr-cloud-provider-check
```

File: testgrid/config.yaml [hunk 2]
```diff
@@ -7804,6 +7813,15 @@ dashboards:
   - name: pr-k8s-e2e
     description: Run e2e tests
     test_group_name: pull-kubernetes-e2e-aks-engine-azure
+  - name: pr-cloud-provider-e2e
+    description: Run Kubernetes e2e tests for cloud-provider-azure
```
Comment (ritazh) [inline testgrid/config.yaml]: Should we also include `pull-cloud-provider-azure-e2e-ccm` here?

Comment (feiskyer) [reply testgrid/config.yaml]: Yep, doing it in another PR #11846.

File: testgrid/config.yaml (hunk 1 above)
Comment (feiskyer) [reply testgrid/config.yaml]: thanks, good catch

**Comment by feiskyer** [comment]
@ritazh @spiffxp Addressed comments, PTAL

File: testgrid/config.yaml (hunk 2 above)
Comment (ritazh) [reply testgrid/config.yaml]: But I missed it, but I do not see the `pull-cloud-provider-azure-e2e-ccm` job under `- name: sig-azure-master` in testgrid/config.yaml https://github.com/kubernetes/test-infra/pull/11846/files#diff-0e9f7582435fdc364e5f7adce048d956R7805

Comment (feiskyer) [reply testgrid/config.yaml]: Oh, missed it. Let me add it back.

**Pull request merged**